jax_distribution/evaluator.py

```diff
@@ -130,22 +135,25 @@ def evaluator_setup(
     network: Any,
     params: FrozenDict,
     config: Dict,
+    centralised: bool = False,
 ) -> Tuple[callable, callable, Tuple]:
-    """Initialise evaluator_fn, network, optimiser, environment and states."""
+    """Initialise evaluator_fn, network, environment and states."""
     # Get available TPU cores.
     n_devices = len(jax.devices())
 
-    # Vmap it over number of agents.
-    vmapped_eval_network_apply_fn = jax.vmap(
-        network.apply,
-        in_axes=(None, 0),
-    )
+    # Vmap it over number of agents based on the type of network.
+    if centralised:
```

Comment:
Very minor suggestion: 
```suggestion
    if centralised_critic:
```